src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Startup.cs

```diff
@@ -26,8 +19,10 @@
 using Microsoft.Extensions.Hosting;
 using System.Linq;
 #if (IndividualLocalAuth)
-using ComponentsWebAssembly_CSharp.Server.Data;
-using ComponentsWebAssembly_CSharp.Server.Models;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+#endif
+#if (GenerateGraph)
```

Comment:
At https://github.com/AzureAD/microsoft-identity-web/blob/master/ProjectTemplates/templates/ComponentsWebAssembly-CSharp/Server/Startup.cs#L23-L25 this logic is nested in IndividualLocalAuth. That looked wrong to me, can you explain why it should be nested?

Reply:
I'm going to assume this was just an oversight and apply what I think is the right fix but would like to still double check if this was intentional @jmprieur @jennyf19

Reply:
Yes, I agree this was wrong; thanks @JunTaoLuo